src/bn_wallets.erl

```diff
@@ -414,3 +430,42 @@ compact_db(#state{db = DB, default = Default, wallets = WalletCF}) ->
     rocksdb:compact_range(DB, Default, undefined, undefined, []),
     rocksdb:compact_range(DB, WalletCF, undefined, undefined, []),
     ok.
+
+-spec jsonrpc_binary_to_token_type(
+    TokenBin :: binary()
+) -> atom().
+jsonrpc_binary_to_token_type(Token) ->
+    case catch binary_to_existing_atom(Token) of
```

Comment:
Maybe wrap this in a standard outer `try/catch` so we can return the error to the caller and tell them what was wrong with the value they supplied and then have an inner case clause that checks the supplied atom is one of those in this list: https://github.com/helium/blockchain-core/blob/b6aca665dfa1cf5bdf81bf4c410275a463b0cc0c/src/blockchain_token_v1.erl#L15?

Reply:
This is just a helper function for JSONRPC users and it reduces the logic required by all callers who use it. I think the idea was to get a MVP here and that it would be pretty easy for the caller to tell they used an unknown token.